.github/workflows/execute-chart-test.yml

```diff
@@ -0,0 +1,33 @@
+name: chart-test
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'releng/**'
+      - 'charts/**'
```

Comment:
probably shouldn't ignore the charts directory on charts test lol
This could likely be simplified to just only run this on changes to `charts/**`

Reply:
Oops!